index.js

```diff
@@ -362,6 +364,14 @@
         }
       });
 
+  //  keys :: StrMap a -> Array String
+  var keys = function(strMap) {
+    if (typeof Object.keys === 'function') return Object.keys(strMap);
```

Comment:
> That said, how desirable is ES3 support?

Dropping support for ES3 seems popular. I'm happy to do so, though I'd like to add a section to the readme to document which environments we target. The description could be as brief as _Sanctuary is designed to work in Node and in ES5-compatible browsers._